das/metrics.go

```diff
@@ -24,7 +24,9 @@ type metrics struct {
 	sampleTime    syncfloat64.Histogram
 	getHeaderTime syncfloat64.Histogram
 	newHead       syncint64.Counter
-	lastSampledTS int64
+
+	lastSampledTS   int64
```

Comment:
fwiw this could also be uint64 for consistency

Reply:
noted, (_although not related to this PR_)